plugin/core/open.py

```diff
@@ -86,8 +87,13 @@ def open_file(
     if view and _return_existing_view(flags, window.get_view_index(view)[0], window.active_group(), group):
         return Promise.resolve(view)
 
+    was_already_open = view is not None
     view = window.open_file(file, flags, group)
     if not view.is_loading():
+        if was_already_open and (flags & sublime.SEMI_TRANSIENT):
+            # work-around bug https://github.com/sublimehq/sublime_text/issues/2411 where transient view might not get
+            # its view listeners initialized.
+            sublime_plugin.check_view_event_listeners(view)  # type: ignore
```

Comment:
I tested this.
This needs to exist, if not the server will not start in the side by side view. 

@rchl I don't know how you managed to find check_view_event_listeners, but it works :)

Reply:
deathaxe found this out in the core issue